Makefile

```diff
@@ -10,9 +10,19 @@ build.watch:
 compile_translations:
 	cd edx_credentials_themes && django-admin.py compilemessages
 
+detect_changed_source_translations:
+	cd edx_credentials_themes && i18n_tool changed
+
+dummy_translations:
+	cd edx_credentials_themes && i18n_tool dummy
+
+ensure_translations: generate_translations detect_changed_source_translations validate_translations
+
 extract_translations:
 	cd edx_credentials_themes && django-admin.py makemessages -l en -d django
```

Comment:
Why not use `i18n_tool` for this? See https://github.com/edx/course-discovery/blob/3900dda2ae0fd6cacf9d8b012319e76856a93617/Makefile#L65-L68.

Reply:
`i18n_tool extract` expects there to be javascript files with strings for translation, and fails if there are none (ie, when `django-admin.py makemessages` doesn't produce a djangojs.po file). This make target is almost identical to the credentials target https://github.com/edx/credentials/blob/master/Makefile#L95